source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,471 @@
+VMs attached to those external networks.
+
+The networking API v2 defines security groups as a white list of rules.
+That means, that by default, neither incoming nor outgoing traffic is
+allowed (from the VMs perspective).
+
+A security group is applied to a logical port - a *logical switch port*
+in the OVN model.
+
+### Owner
+
+* Feature Owner: [Miguel Duarte Barroso](https://github.com/maiqueb) (mbarroso on IRC)
+* Email: <mdbarroso@redhat.com>
+
+### Terminology
+
+* Networking API: [OpenStack RESTful Networking API](https://developer.openstack.org/api-ref/network/v2/)
+* OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that
+brings Software Defined Networking to OVS.
+* oVirt OVN provider: a proxy that implements a subset of the Networking API
+and interacts with OVN.
+* Security group: container for security group **rules**; networking API
+entity
+* Security group rules: specify the access to a network resource;
+networking API entity
+
+### Benefit to oVirt
+
+Currently, the only security mechanisms present for oVirt VMs attached to
+external networks are  quite crude L2 and L3 protection - MAC
+spoofing, and IP spoofing - implemented directly by Open vSwitch. It is worth 
+mentioning that IP spoofing protection is only applied when a subnet is defined
+on top of the Networking API network.
+
+Security groups, which are a traffic white list, complement provided security
+by specifying which traffic is allowed to *and* from the resources - e.g. VMs -
+using high level abstraction L3 and L4 semantics.
+
+Furthermore, security groups allow these rules to be applied in fine
+grained fashion - e.g. the user can specify which rules apply to which
+VMs.
+
+Below, you can find some examples of what is the feature's ambition:
+* **only** allow incoming traffic to a specific VM from a specific
+CIDR
+* **only** allow incoming access to a range of destination ports in a
+set of VMs
+* **only** allow outgoing traffic to a specific port on a set of VMs
+
+Having said that, and highlighting that L2 protection is **outside of
+scope** of security groups, it could be further enhanced by implementing the
+[Allowed address pairs](https://developer.openstack.org/api-ref/network/v2/#allowed-address-pairs)
+Networking API extension.
+
+## Objectives
+
+* Provide a security group implementation, using OVN as the backend.
+No additional backend / database should be used other than ovn's
+northbound database.
+
+* When possible, Openstack neutron's behavior should be mimicked: by
+default, all VM incoming IP *and* outgoing traffic is dropped. This behavior
+is better specified in the [requirements](#requirements) section.
+
+* The security group feature should be configurable, and activated by
+default - this means an option for using oVirt's external networks
+without security group support should be possible.
+
+* If the external network has a subnet defined, DHCP traffic is allowed to
+/ from the VM.
```

Comment:
Isn't this the OVN default behaviour.
Seems like a bug to me on OVN if they don't allow this implicitly.